atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransaction.java

```diff
@@ -2260,10 +2269,10 @@ private void handleKeyAlreadyExistsException(
                         + " because our locks timed out. startTs: " + getStartTimestamp() + ".  "
                         + getExpiredLocksErrorString(commitLocksToken, expiredLocks), ex);
             } else {
-                log.debug("Possible bug: Someone rolled back our transaction but"
-                        + " our locks were still valid; Atlas code is not allowed to do this, though others can.",
-                        SafeArg.of("immutableTimestampLock", immutableTimestampLock),
-                        SafeArg.of("commitLocksToken", commitLocksToken));
+                log.info("This transaction has been rolled back by someone else.",
+                        SafeArg.of("immutableTimestampLock",
```

Comment:
@gmaretic would you be able to quickly test manually using an internal product to see that this logline works?